### DISCUSSION TITLE
problem in style of @fluentui/react-datepicker-compat & @fluentui/react-timepicker-compat

### DISCUSSION THREAD

**Discussion by abolfathzadeh** [microsoft/fluentui#30661] [Q&A]
I am using a date picker and time picker  in my project but the style that is now applied, however, the style of other components of the Fluent UI don't have any problem:
![image](https://github.com/microsoft/fluentui/assets/9671082/6d9b4cd9-bdc1-45be-aa6e-311e0c59dd11)
the result in UI is: 
![image](https://github.com/microsoft/fluentui/assets/9671082/12077b6d-bbe3-46c2-b9f9-f0df5dce9a45)
as it is shown: the top one, which is a Calender, shows perfectly but the other in buttom which is a date picker, does not have any style, I checked this date picker and time picker in any part of the project, and still have style problem.

**Comment by khmakoto** [comment]
@abolfathzadeh could you check in your lock file if you have more than one version of `@fluentui/react-shared-contexts` existing? If you do, could you deduplicate them and check if that fixes your issue?
